flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -486,4 +630,14 @@ enum FlxAxes
 {
 	X;
 	Y;
+}
+
+enum GamepadModel
+{
+	Logitech;
+	OUYA;
+	PS3;
+	PS4;
+	Xbox;
```

Comment:
Also, I'm not sure how `XInput` fits into this - my understanding is that controllers *are* not really "XInput", they only *use* it (or don't). Right?